docker-ubuntu.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+#Exit on error
+set -e
+
+sudo -v
+
```

Comment:
I highly doubt installation may fail due to missing dependency. Have you tried it on your machine?
check this https://docs.docker.com/engine/installation/linux/docker-ce/ubuntu/#Install using the repository

Reply:
Yes, I have checked the whole script by running it on a separate instance to install Docker CE. It was 100% successful with no errors. Do you specifically notice any missing part?

Reply:
Okay cool then. Just add the verification and remove k8s script from this PR. Add k8s to a different PR.